news/2024.html

```diff
@@ -30,6 +30,42 @@
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="sponsor24-proofcraft">
+  <a href="#sponsor24-proofcraft"><span class="tag-span"># </span></a> 24 May 2024: Thanks Proofcraft for being a Bronze Sponsor of the seL4 Summit 2024
+</div>
+<div>
+   <a href="https://proofcraft.systems">
+    <img src="/Foundation/Membership/LOGOS/proofcraft.svg"
```

Comment:
Links should now all be made relative so that the preview keeps working. (Haven't gotten around to documenting that anywhere yet, sorry)

```suggestion
   <a href="https://proofcraft.systems">
    <img src="../Foundation/Membership/LOGOS/proofcraft.svg"
```